include/power/power_state.h

```diff
@@ -162,28 +162,30 @@ struct pm_state_info {
  *		cpu0: cpu@0 {
  *			device_type = "cpu";
  *			...
- *			pm-states = <&state0 &state1>;
+ *			cpu-idle-states = <&state0 &state1>;
  *		};
  *	};
  *
  *	...
- *	state0: state0 {
- *		compatible = "pm-state";
- *		pm-state = "suspend-to-idle";
- *		min-residency-us = <1>;
- *	};
+ *      idle-states {
+ *		state0: state0 {
+ *			compatible = "zephyr,idle-state";
+ *			idle-pm-state-name = "suspend-to-idle";
+ *			min-residency-us = <1>;
+ *		};
  *
- *	state1: state1 {
- *		compatible = "pm-state";
- *		pm-state = "suspend-to-ram";
- *		min-residency-us = <5>;
+ *		state1: state1 {
+ *			compatible = "zephyr,idle-state";
+ *			idle-state-name = "suspend-to-ram";
+ *			min-residency-us = <5>;
```

Comment:
Are these residency times realistic?  I find it hard to believe the combined entry and exit times for suspend-to-ram would be as tiny as 5 u.

Reply:
these are just arbitrary values used as example for the documentation. These value are target specific and anyone copying from here would be doing a mistake .... Nevertheless this is already committed.